### DISCUSSION TITLE
Query items with multiple counts on related table.

### DISCUSSION THREAD

**Discussion by volkerrichert** [prisma/prisma#17676] [Q&A]
Hi! 

I enabled "filteredRelationCount" and like to receive the number of rows in the related table based on conditions.
``
something like 
```
findMany({
  include: {
    _count: {
      select: {
        items: { where: { status: 'FAILED' } },
      },
    },
  },
})
```

works fine. It will be translated into a left join. But i'd like to have a count for multiple STATUS in the related item table, like so:
```
findMany({
  include: {
    _count: {
      select: {
        items: [
          { where: { status: 'FAILED' } },
          { where: { status: 'SUCCESSFUL' } },
          { where: { status: 'SOME_OTHER_STATUS' } }
        ]
      },
    },
  },
})
```
Is there a way in implement this? Or is using queryRaw the only way?

THX,

Volker

**Comment by nurul3101** [comment]
Hey @volkerrichert 👋 

You would need to repeat the findMany query three times with the individual status, you can wrap them in a [transaction](https://www.prisma.io/docs/concepts/components/prisma-client/transactions).
But it's not possible to have multiple counts in a single query.

We have a feature request here:
- https://github.com/prisma/prisma/issues/15423

The other alternative is to use [$queryRaw](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) as you suggested.
Also, if you can add a 👍 to the feature request then it would help our product team in prioritising it.

**Reply by volkerrichert** [reply]
I'll use raw query with Prisma.sql ATM

THX

**Reply by nurul3101** [reply]
No worries! 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
